changelog/646.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+- Adds `None` check on `self.aligned_axes` to `NDCollection.copy()` and `NDCollection.pop()` which raised a `AttributeError`
```

Comment:
We try to keep our changelog user facing, so a simpler description of what the bug fixed was. 

No need to go into much detail about specifics or tests added. 

Thanks!